src/Twig.php

```diff
@@ -100,7 +105,7 @@ public function fetchBlock($template, $block, $data = [])
     {
         $data = array_merge($this->defaultVariables, $data);
 
-        return $this->environment->loadTemplate($template)->renderBlock($block, $data);
+        return $this->environment->load($template)->renderBlock($block, $data);
```

Comment:
Available in Twig 1. See https://github.com/twigphp/Twig/blob/1.x/src/Environment.php#L413